<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>764</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #31363b;
color: rgb(200, 200, 200);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="top_frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40,40,40);

</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="top_label">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="frame_label_top_btn_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="toplabel_icon">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-image: url(styles/icons/16x16/cil-chart-line.png);
background-position: center;
background-repeat: no-repeat;
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="toplabel_title">
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
             <underline>false</underline>
             <strikeout>false</strikeout>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">margin-left: 3px;</string>
           </property>
           <property name="text">
            <string>ShowMeTheCoin v1.0</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="openExternalLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_btn_right">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="frame_btn_2" stretch="0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="minimize_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Minimize</string>
           </property>
           <property name="statusTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="accessibleName">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {	
	border: none;
	background-color: transparent;
}
QPushButton:hover {
	background-color: rgb(52, 59, 72);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/20x20/cil-window-minimize.png</normaloff>../icons/20x20/cil-window-minimize.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="maximize_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Maximize</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {	
	border: none;
	background-color: transparent;
}
QPushButton:hover {
	background-color: rgb(52, 59, 72);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/16x16/cil-window-maximize.png</normaloff>../icons/16x16/cil-window-maximize.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close_btn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Close</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {	
	border: none;
	background-color: transparent;
}
QPushButton:hover {
	background-color: rgb(52, 59, 72);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/20x20/cil-x.png</normaloff>../icons/20x20/cil-x.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="content">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QFrame" name="login_area">
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>550</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QFrame" name="avatar">
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>130</y>
           <width>121</width>
           <height>121</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame {
	border-radius: 60px;
	border: 10px solid rgb(85, 170, 255);
	background-position: center;
}

</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>4</x>
            <y>0</y>
            <width>117</width>
            <height>121</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>45</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color : rgb(85, 170, 255);
border: 0px solid rgb(85, 170, 255);
background-position: center;
background: transparent;</string>
          </property>
          <property name="text">
           <string>₿</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
        <widget class="QLineEdit" name="lineEdit_access">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>290</y>
           <width>271</width>
           <height>50</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);	
	color: rgb(100, 100, 100);
}

QLineEdit:focus {
	border: 2px solid rgb(85, 170, 255);
	color: rgb(200, 200, 200);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="placeholderText">
          <string>Access Key</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_secret">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>360</y>
           <width>271</width>
           <height>50</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit {
	border: 2px solid rgb(45, 45, 45);
	border-radius: 5px;
	padding: 15px;
	background-color: rgb(30, 30, 30);	
	color: rgb(100, 100, 100);
}

QLineEdit:focus {
	border: 2px solid rgb(85, 170, 255);
	color: rgb(200, 200, 200);
}</string>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Secret Key</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox_save_user">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>440</y>
           <width>281</width>
           <height>22</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QCheckBox::indicator {
	border: 3px solid rgb(100, 100, 100);
	width: 15px;
	height: 15px;
	border-radius: 10px;	
	background-color: rgb(135, 135, 135);
}
QCheckBox::indicator:checked {
	border: 3px solid rgb(85, 170, 255);
	background-color: rgb(85, 120, 255);
}</string>
         </property>
         <property name="text">
          <string>SAVE USER</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_connect">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>490</y>
           <width>271</width>
           <height>50</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {	
	background-color: rgb(30, 30, 30);
	border: 2px solid rgb(60, 60, 60);
	border-radius: 5px;
}
QPushButton:hover {	
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(70, 70, 70);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
	border: 2px solid rgb(85, 170, 255);
	color: rgb(35, 35, 35);
}</string>
         </property>
         <property name="text">
          <string>CONNECT</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>10</y>
           <width>361</width>
           <height>101</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Masque</family>
           <pointsize>40</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color : rgb(85, 170, 255);</string>
         </property>
         <property name="text">
          <string>SIGN IN</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="bottom">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(15, 15, 15)</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_credits">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(75, 75, 75);</string>
        </property>
        <property name="text">
         <string>Created by: Wanderson M. Pimenta</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
